bentoml/handlers/base_handlers.py

```diff
@@ -81,4 +83,4 @@ def get_output_str(result, output_format, output_orient="records"):
                 # when result is not JSON serializable
                 return json.dumps(str(result))
     else:
-        raise ValueError("Output format {} is not supported".format(output_format))
+        raise BadRequest("Output format {} is not supported".format(output_format))
```

Comment:
Can we define our own BadInput exception so it can be properly handled when loading a BentoServiceAPI for use with CLI or AWS Lambda?